sentry_sdk/integrations/asgi.py

```diff
@@ -21,11 +27,15 @@
     from typing import Optional
     from typing import Callable
 
+    from typing_extensions import Literal
+
     from sentry_sdk._types import Event, Hint
 
 
 _asgi_middleware_applied = ContextVar("sentry_asgi_middleware_applied")
 
+TRANSACTION_SENTINEL = "generic ASGI request"
```

Comment:
There is a possibilty this value will be the final one in case of errors that are not attributable to a particular route (errors happening during routing for example).

I can rename it to `_DEFAULT_TRANSACTION_NAME`, does that make sense?

Reply:
`_DEFAULT_TRANSACTION_NAME` or `DEFAULT_TRANSACTION_NAME` sgtm. Public in this case would signal that other middleware could reference it (e.g. to do the same as we do here -- only change if it is the default).

In terms of ergonomics, you may also consider whether a class attribute would make more sense, such that you only need to import `SentryAsgiMiddleware` and you have the constant there as  
`SentryAsgiMiddleware.[_]DEFAULT_TRANSACTION_NAME`.

Reply:
I'll keep this as global since people are not supposed to override it, they can do the same using our regular scope api